spec/controllers/static_controller_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe StaticController, type: :controller do
+  render_views
+
+  describe "GET home" do
+    it "renders the home template with the default layout" do
+      get :home
+      expect(response).to render_template("home")
+      expect(response).to render_template("layouts/application")
+
+      assert_select "h1", "Flatiron Widgets Store"
```

Comment:
We use capybara style view specs. So something like this https://github.com/learn-co-curriculum/validations-with-form_tag-rails-lab/blob/master/spec/views/posts/edit.html.erb_spec.rb

Students won't know `assert_select`